### DISCUSSION TITLE
.Net Framework 4.7.2

### DISCUSSION THREAD

**Discussion by aggiczy** [Hawxy/Discord.Addons.Hosting#17] [Q&A] [answered]
Hi! Can I use the new version (4.0.0) of Discord.Addons.Hosting with .Net Framework 4.7.2?

**Comment by Hawxy** [comment, marked as answer]
No sorry. I have never officially supported .NET Framework, and the last pieces of compatibility were removed back in November last year.

Unless there's a very specific reason to be on .NET Framework, I'd highly recommend moving to .NET 5.

**Reply by aggiczy** [reply]
Okay, thank you for the information.
